3-UsingAPIs/Week1/homework/ex4-pokerDiceAll.js

```diff
@@ -27,9 +27,12 @@ exercise file.
 const rollDie = require('../../helpers/pokerDiceRoller');
 
 function rollDice() {
-  // TODO Refactor this function
   const dice = [1, 2, 3, 4, 5];
-  return rollDie(1);
+  return Promise.all(
+    dice.map((item) => {
+      return rollDie(item);
+    })
+  );
```

Comment:
You can shorten this to:

```suggestion
  return Promise.all(dice.map(rollDie));
```

Can you figure out why this would work too? Of course, your version is fine also.

Reply:
Yeap, because rollDie accept only one parameter which is passed by map function